chapter_5/chapter_5.md

```diff
@@ -0,0 +1,32 @@
+## Chapter 5
+- Since most correlations do not indicate causal relationships, we need tools for distinguishing mere association from evidence of causation. This is why so much effort is devoted to **multiple regression**, using more than one predictor variable to simultaneously model an outcome. Reasons given for multiple regression models include:
+    - To emulate multiple and complex causation
+    - To model interactions: The importance of one variable may depend upon another. Such interactions occur very often. Effective inference about one variable will often depend upon consideration of others.
+-   The general formulae for a multiple linear regression features a new relation between the mean of the Gaussian distribution and the predictor variables $x_{j}$:
```

Comment:
formulae (noun) is plural but features (verb) is singular? Maybe it should say "formula"